index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css">
+    <title>Review Past Project</title>
+  </head>
+
+  <body>
+    <main id="main-doc">
+      <div class="box">
+        <nav id="navbar">
+          <header class="h1">JS Challenges</header>
+          <a href="#Apple_and_Orange" class="nav-link">Apple and Orange</a>
+          <a href="#Breaking_best_and_worst_records" class="nav-link"
+            >Breaking best and worst records</a
+          >
+          <a href="#Cats_and_a_mouse" class="nav-link">Cats and a mouse</a>
+          <a href="#The_hurdle_race" class="nav-link">The hurdle race</a>
+          <a href="#Simple_Array_Sum" class="nav-link">Simple Array Sum</a>
+        </nav>
+      </div>
+
+      <div class="container">
```

Comment:
- [ ] Also, I would suggest using the `<main>` tag to contain the "main" content below the `<header>` tag, instead of the `<div>` with `class="container"`. You can see information about it [here](http://html5doctor.com/the-main-element/).

![image](https://user-images.githubusercontent.com/81522435/135642712-ef8f5589-d534-403b-833b-5c4de994a416.png)